fbfmaproom/fbf-update-data.py

```diff
@@ -341,6 +341,34 @@
         "niger/wrsi-jas",
         "http://iridl.ldeo.columbia.edu/SOURCES/.USGS/.EROS/.FEWS/.dekadal/.WAF/.Millet/.do/X/0.125/15.875/RANGE/Y/11.125/16.875/RANGE/T/(Jul-Sep)/seasonalAverage/T/(months%20since%201960-01-01)/streamgridunitconvert/",
     ),
+    (
+        'mali/chirps-precip-jun',
```

Comment:
Please name this `mali/chirps-precip-jun-jas`, to indicate that the T values have been faked (shiftGRID) to make it look like JAS. See e.g. https://github.com/iridl/python-maprooms/blob/master/fbfmaproom/fbfmaproom-sample.yaml#L885 for example.

Likewise for the other time-shifted datasets below.